test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -4208,46 +4241,77 @@ def _load_configuration(self, infile, objtype=None):
         if os.path.isfile(infile):
             conf = {}
             with open(infile, 'r') as f:
-                # load all objects from the Pickled file
-                while True:
-                    try:
-                        conf = cPickle.load(f)
-                    except:
-                        break
-
-            if objtype and objtype in conf:
-                conf = conf[objtype]
-            else:
-                # load all object types that could be in infile
-                newconf = {}
-                for ky in [MGR_OBJ_SERVER, MGR_OBJ_SCHED, MGR_OBJ_NODE]:
-                    if ky not in conf:
-                        conf[ky] = {}
-                    newconf = dict(newconf.items() + conf[ky].items())
-                conf = newconf
-
+                try:
+                    conf = json.load(f)
+                except ValueError:
+                    self.logger.info("Error loading JSON file: %s"
+                                     % infile)
+            qmgr = os.path.join(self.client_conf['PBS_EXEC'],
+                                'bin', 'qmgr')
             for k, v in conf.items():
-                fn = self.du.create_temp_file()
-                with open(fn, 'w') as fd:
-                    # handle server data saved as output of qmgr commands
-                    # by piping data back into qmgr
-                    if k.startswith('qmgr_'):
-                        qmgr = os.path.join(self.client_conf['PBS_EXEC'],
-                                            'bin', 'qmgr')
-                        fd.write("\n".join(v))
-                        self.du.run_cmd(self.hostname, [qmgr], cstdin=fd,
-                                        sudo=True)
-                    else:
-                        fd.write("\n".join(v))
-                        # append the last line
-                        fd.write("\n")
-                        self.du.run_cmd(self.hostname, ['cp', fn, k],
-                                        sudo=True)
-                    os.remove(fn)
-
+                # Load server configuration
+                if k.startswith('qmgr_'):
+                    fpath = self.du.create_temp_file()
+                    print_svr = '\n'.join(v)
```

Comment:
Are we just loading server here or all objects? Then I think we should update the docstring accordingly.